ci/test/resources/build.yaml

```diff
@@ -54,6 +68,22 @@ steps:
      valueFrom: default_ns.name
    dependsOn:
     - default_ns
+ - kind: createDatabase2
+   name: hello2_database
+   databaseName: hello2
+   migrations:
+    - name: create-tables
+      script: /io/sql/create-hello2-tables.sql
+    - name: insert
+      script: /io/sql/insert.py
+   inputs:
+    - from: /repo/ci/test/resources/sql
+      to: /io/
+   namespace:
+     valueFrom: default_ns.name
+   dependsOn:
+    - default_ns
```

Comment:
Should we add a test to `test_ci.py` that verifies the insert happened properly? Not sure how hard that would be. You'd have to make deploy_hello dependent on hello2_database.

Reply:
Totally agree.  Instead of mixing it with deploy_hello, I added a separate test_hello2_database step that just verifies the migration version is 3.